src/ui/components/UploadScreen/TeamSelect.tsx

```diff
@@ -36,7 +36,7 @@ export default function TeamSelect({
   let displayedWorkspaces: DisplayedWorkspace[] = [...workspaces].sort();
 
   if (userInfo.features.library) {
-    displayedWorkspaces = [personalWorkspace, ...displayedWorkspaces];
+    displayedWorkspaces = [personalWorkspace];
```

Comment:
:question: What's the change here in relation to the rest?

Reply:
GOOD CALL, this was my test code. This is why reviews are nice :)